### ISSUE TITLE
`Windows.System.Pslist` errors "Symbol token not found" when using Windows x86 collector

### ISSUE THREAD

**Issue by hunt3vil** [Velocidex/velociraptor#3627]
I am experiencing some issues with Windows x86 offline collectors when running Windows.System.Pslist which I don't understand and don't know where to start troubleshooting.

I am getting a large number of these errors:

```
Symbol token not found. Current Scope is [NULL], [_SessionId, $device_manager, config, $acl, Artifact, $repository, $root], [$Query], [$acl, config, $root, $repository, $stack], [ProcessRegex, PidRegex, ExePathRegex, CommandLineRegex, UsernameRegex, UntrustedAuthenticode, UseTracker, DISABLE_DANGEROUS_API_CALLS], [UntrustedAuthenticode], [UseTracker], [DISABLE_DANGEROUS_API_CALLS], [ProcList], [Windows_System_Pslist_0_1], [$Query], [Pid, Ppid, Name, Threads, Username, OwnerSid, CommandLine, Exe, TokenIsElevated, CreateTime, User, System, IoCounters, Memory, PebBaseAddress, IsWow64]
```

**Comment by predictiple** [comment]
Some VQL functions are not supported under x86 and token() is one of them: https://github.com/Velocidex/velociraptor/blob/master/vql/windows/process/token.go

**Comment by scudette** [comment]
Maybe we should add noop plugins to silence the warnings?

**Comment by hunt3vil** [comment]
Something like that. I just created a custom artifact with an `if` condition based on architecture but would rather not maintain my own for very long as I also have to rewrite the NetstatEnriched artifact to call my custom artifact.

**Comment by scudette** [comment]
This is fixed with #3643
